ragna/deploy/_ui/components/metadata_filters_builder.py

```diff
@@ -0,0 +1,307 @@
+from datetime import datetime
+
+import panel as pn
+import param
+
+from ragna.core._metadata_filter import MetadataFilter
+
+METADATA_FILTERS = {
+    "document_name": {"type": str},
+    "size": {"type": int},
+    "extension": {"type": str},
+    "section": {"type": str},
+    # "creation_date": {"type": datetime},
+}
+
+
+FILTERS_PER_TYPE = {
+    str: ["==", "!=", "in", "not in"],
+    int: ["==", "!=", ">", "<", ">=", "<=", "in", "not in"],
+    # datetime: ["==", "!=", ">", "<", ">=", "<="],
+}
+
+PLACEHOLDERS_PER_METADATA_KEY = {
+    "document_name": "",
+    "size": "",
+    "extension": "",
+    "section": "",
+    # "creation_date": "YYYY-mm-dd HH:MM:SS",
+}
```

Comment:
Do we need this at all for now? Looks like all placeholders are empty at the current stage.

Reply:
Placeholders make a lot of sense for the datetimes, but we don't have any dt metadata anymore, so we don't really need them for the moment. Should I remove everything related to placeholders ?

Reply:
I think so, yes. Datetimes would be a nice feature, but until we have decided how we want to deal with them, let's shed dead code.